releasenotes/notes/v05-encoding-pack-str-error-handle-d11960b72472af7c.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an issue where using the v0.5 encoding endpoint did not raise any ``TypeError``
```

```suggestion
    tracing: Resolves an issue where using the v0.5 encoding endpoint did not raise any ``TypeError``
```

is "This fix" redundant given it's in the fixes section?